assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -146,12 +149,35 @@ function createResourceAccessChart ({ data, width, height, domElement }) {
   const miniYScale = d3.scaleBand().range([0, miniHeight])
 
   const textScale = d3.scaleLinear().range([12, 6]).domain([15, 50]).clamp(true)
+  const unReadResources = resourceData.filter(resource => resource.self_access_count === 0)
+  // d3 graph narrative text
+  const narrativeTextResources = {}
+  narrativeTextResources.weekRange = `Resources accessed from week ${weekRange[0]} to ${weekRange[1]}.  `
+  narrativeTextResources.resourceType = `Selected resource type is ${resourceType}.  `
+  narrativeTextResources.gradeFilter = gradeSelection.toUpperCase() !== 'ALL' ? `Filtering on grades ${gradeSelection}.  ` : ''
+  narrativeTextResources.resourcesUnaccessCount = `You have not yet accessed ${unReadResources.length} of ${resourceData.length} resources.  `
+  narrativeTextResources.resourcesList = resourceData.map(x =>
+    `${x.resource_name.split('|')[1]} of type ${x.resource_type} accessed ${x.total_percent}% and ${resourceToolTipText(x, false)} `
```

Comment:
If we only have one type in the course or one type selected it's redundant to keep saying the resource_type every time. I feel like most classes will only have one time. This should be dropped out.

Reply:
As a user I think it would be best if it had

The resourcesListUnaccessed
Then after the resourcesListAccessed

as two separate lists. It could read off all of the unaccssed entries then the accessed. I'm thinking of

This is good
`"Resources accessed from week 6 to 8.  Selected resource type is Files.  You have not yet accessed 4 of 36 resources."`

Then after
`<List of unacgessed resources without the resourceToolTipText, since it just says "You haven't accessed this resource">`

Then after that it could have
`"You have accessed 32 of 36 resources"`

And have the rest of the list of accessed resources. We might want to have a briefer resourceToolTipText for these cases?

Reply:
I have separated out the access vs unaccess narrative. Also I am not appending resource type if that is the selected type